library/core/class.configuration.php

```diff
@@ -376,6 +376,9 @@ public function get($name, $defaultValue = false) {
             if (is_array($value) && array_key_exists($keys[$i], $value)) {
                 $value = $value[$keys[$i]];
             } else {
+                if ($this->Data["TranslationDebug"] ?? false) {
```

Comment:
This looks like the right place, but I think we could give it a better name.

Maybe we can just make a private property + setter & call it the container.

```
                if ($this->fallbackDecorator ?? false) {
                	$defaultValue = $this->fallbackDecorator . $defaultValue . $this->fallbackDecorator;
                }
```